external/libefp/CMakeLists.txt

```diff
@@ -28,7 +28,8 @@ else()
                    -DLIBC_INTERJECT=${LIBC_INTERJECT}
                    -DFRAGLIB_UNDERSCORE_L=OFF
                    -DFRAGLIB_DEEP=OFF
-        CMAKE_CACHE_ARGS -DCMAKE_C_FLAGS:STRING=${CMAKE_C_FLAGS}
+                   -DLAPACK_LIBRARIES=${LAPACK_LIBRARIES} 
```

Comment:
Yeah, I was relying on libefp.a blas symbols being satisfied by psi4's at link-time, so probably good to pass in directly. Will look into whether `LAPACK_LIBRARIES` or `LAPACK_INTERJECT` is more appropriate.